dsnp/broadcast.ts

```diff
@@ -0,0 +1,41 @@
+import { ParquetModel } from "../types/frequency";
+
+const broadcast: ParquetModel = [
+  {
+    // TODO: Is this needed?
+    name: "announcementType",
+    column_type: {
+      INTEGER: {
+        bit_width: 32,
+        sign: true,
```

Comment:
We should document somewhere that announcementType <0 is invalid, and that this is here because Parquet [doesn't support unsigned ints](https://parquet.apache.org/docs/file-format/types/).

Reply:
The logical types do, so I should swap this over to 16 unsigned.

That said really it's a constant, but parquet doesn't have a way to handle that.